oneview_redfish_toolkit/services/manager_service.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (2018) Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+# Own libs
+from oneview_redfish_toolkit import multiple_oneview
+
+
+def get_manager_uuid(resource_id):
+    map_resources = multiple_oneview.get_map_resources()
+    ov_ip = map_resources[resource_id]
```

Comment:
@adriane-cardozo we don't need to do this check because this method is only called after the resource_id filled in cache, as you can see on this [example](https://github.com/HewlettPackard/oneview-redfish-toolkit/blob/d34afd3ded5a3ef49443ab340426b4a159191038/oneview_redfish_toolkit/blueprints/chassis.py#L57). Here the uuid request is already mapped, so I can search for it on cache, as happens on other calls like this.

Reply:
Got it!